ncov2019ArticNf.wdl

```diff
@@ -30,8 +30,8 @@ workflow ncov2019ArticNf {
     File trimPrimerSequencesBam = illumina_ncov2019ArticNf.trimPrimerSequencesBam
     File trimPrimerSequencesPrimerTrimmedBam = illumina_ncov2019ArticNf.trimPrimerSequencesPrimerTrimmedBam
     File makeConsensusFasta = illumina_ncov2019ArticNf.makeConsensusFasta
+	File makeConsensusVcf = illumina_ncov2019ArticNf.makeConsensusVcf
```

Comment:
Weird indentation on this line. I think the file is indented with spaces.

Reply:
fixed that